packages/documentation/src/stories/getting-started/components-angular/angular.docs.mdx

```diff
@@ -0,0 +1,26 @@
+import { Meta, Source } from '@storybook/blocks';
+import * as GettingStartedStories from './angular.stories';
+import AngularAppModuleSample from './angular-app-module.sample.ts?raw';
+import AngularAppComponentSample from './angular-app-component.sample.html?raw';
+
+<Meta of={GettingStartedStories} />
+
+# @swisspost/design-system-components-angular
+
+[![npm version](https://badge.fury.io/js/@swisspost%2Fdesign-system-components-angular.svg)](https://badge.fury.io/js/@swisspost%2Fdesign-system-components-angular)
+
+A set of generated wrapper components around our web-components, for easy integration with the Angular framework.
```

Comment:
Not sure if this is a selling point. I'd just talk about Angular components.
```suggestion
The Design System Angular Components, ready for use in your Angular project.
```

Reply:
In my opinion, it's worth it to be honest and keep mentioning that there are only wrappers around web-components and not Angular components made manually.